src/components/date/date.stories.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import {
+  boolean, number, text, select
+} from '@storybook/addon-knobs';
+import OptionsHelper from '../../utils/helpers/options-helper';
+import notes from './notes.md';
+import DateInput from './date.js';
+
+storiesOf('DateInput', module)
+  .add('default', () => {
+    const autoFocus = boolean('autoFocus', true);
+    const disabled = boolean('disabled', false);
+    const minDate = text('minDate', '');
+    const maxDate = text('maxDate', '');
+    const readOnly = boolean('readOnly', false);
+    const label = text('label', 'Example DateInput');
+    const labelWidth = number('labelWidth', 0);
+    const labelHelp = text('labelHelp', 'This text provides more information for the label.');
+    const fieldHelp = text('fieldHelp', 'This text provides help for the input.');
+    const fieldHelpInline = boolean('fieldHelpInline', false);
+    const value = text('value', DateInput.defaultProps.value);
+    const labelInline = boolean('labelInline', false);
+    const inputWidth = labelInline ? number('inputWidth', 0, {
+      range: true,
+      min: 0,
+      max: 50,
+      step: 1
+    }) : undefined;
+    const labelAlign = labelInline ? select(
+      'labelAlign',
+      OptionsHelper.alignBinary,
+      OptionsHelper.alignBinary[0]
+    ) : undefined;
+
+    return (
```

Comment:
We shouldn't see this.

![image](https://user-images.githubusercontent.com/26925206/53880588-43ed0180-4009-11e9-8bd8-bd7cd310c3be.png)

Reply:
Currently it is generated value. It's possible to change default prop table. We can define custom PropTable based on https://github.com/storybooks/storybook/blob/master/addons/info/src/components/PropTable.js and using `TableComponent` parameter for addon-info. Maybe it is good time to create ticket for it and try to do it in the next sprint. What do you think?